### ISSUE TITLE
Adjust Y position of text when it gets smaller

### ISSUE THREAD

**Issue by CameronSamuels** [thecoderators/memerator#62]
When the text gets smaller, it looks lower.

**Comment by Pipe-to-Grep** [comment]
Right now, it looks like the text is constricted to a set x range. It can only get so big without resizing. It needs to be set to an overflow.

**Comment by CameronSamuels** [comment]
Yes, @Pipe-to-Grep! We need to make a dynamic variable (changes) instead of a static variable (the same).

We can make it have a relation to the height of the text. Currently we have the y-position be related to a specific value of what we previously had the height of the text to be (when it was always the same).
